include/miopen/miopen.h

```diff
@@ -6582,6 +6584,73 @@ MIOPEN_EXPORT miopenStatus_t miopenBackendInitialize(miopenBackendDescriptor_t d
 // CLOSEOUT BackendAPI DOXYGEN GROUP
 #endif // MIOPEN_BETA_API
 
+#ifdef MIOPEN_BETA_API
+
+// Diagonal APIs
+/** @addtogroup diagonal
+ *
+ *  @{
+ */
+
+/*! @brief Execute Diag forward layer
+ *
+ * @param handle               MIOpen handle (input)
+ * @param inputDesc            Tensor descriptor for input tensor (input)
+ * @param input                Input tensor (input)
+ * @param outputDesc           Tensor descriptor for output tensor (input)
+ * @param output               Output tensor (output)
+ * @param diagonal             Which diagonal to consider (input)
```

Comment:
The document should contain info about the default value of `diagonal` param.

Reply:
solved